database/schema.sql

```diff
@@ -2,6 +2,7 @@ CREATE TABLE certificates(
 	id	        				serial primary key,
 	sha1_fingerprint	        varchar NOT NULL,
 	sha256_fingerprint	    	varchar NOT NULL,
+    sha256_subject_spki         varchar NOT NULL,
```

Comment:
Looks like there's a tabs/spacees issue here (unless it's just a display issue in github...)